pyfar/plot/line.py

```diff
@@ -358,6 +362,12 @@ def time_freq(signal, dB_time=False, dB_freq=True, log_prefix_time=20,
         ``style = {'axes.facecolor':'black'}``. Pass an empty dictionary
         ``style = {}`` to use the currently active plotstyle. The default is
         ``light``.
+    mode : str
+        ``real``, ``imag``, or ``abs`` to specify if the real part, imaginary
+        part or absolute value of a complex-valued time domain signal is
+        plotted. The default is ``real``.
```

Comment:
I would mention that imag and abs can not be used for non-complex Signals here and everywhere else:

```suggestion
    mode : str
        ``'real'``, ``'imag'``, or ``'abs'`` to specify if the real part, imaginary
        part or absolute value of the time data is plotted. ``'imag'`` and ``'abs'```
        can only be used for complex Signals. The default is ``'real'``.
```